src/renderers/common/Renderer.js

```diff
@@ -1855,6 +1855,8 @@ class Renderer {
 
 		}
 
+		renderContext.clearColorValue = this._clearColor;
```

Comment:
Do you mind explaining why this line is necessary? Besides, only the clear color is copied but what about `this._clearStencil` and `this._clearDepth`?

Reply:
In `RenderContext.js`, the renderContext is always enabled by default with `this.clearColorValue = { r: 0, g: 0, b: 0, a: 1 };`, making the `clear` method of the `WebGLBackend` always using that default value as the value is never getting updated when using `renderer.clear()`.

clearStencil and clearDepth are 2 differents things and don't have the issue:

In Renderer:
```js
this.backend.clear( color, depth, stencil, renderContext );
```

in WebGLBackend:
```js
if ( color ) clear |= gl.COLOR_BUFFER_BIT;
if ( depth ) clear |= gl.DEPTH_BUFFER_BIT;
if ( stencil ) clear |= gl.STENCIL_BUFFER_BIT;
```

Reply:
Okay, understood! Do you mind adding `// #30329` as a comment next to the statement so it's easier to find this discussion?

Ideally, we can refactor the clear related logic a bit so the WebGPU and WebGL backend rely on the same inputs.

Reply:
I think we can use just `getClearColor()` like `WebGPUBackend` removing this first conditional of ` descriptor.clearColorValue`, we can change both if necessary in the future following the same logic.

Reply:
Make sense, updated accordingly, it seems to work fine too from my tests.

Reply:
For you information I had to revert the getClearColor version as it was breaking some examples. My guess is that it breaks when using Background.

I will think about cleaning up the logic in another PR. /cc @sunag

Reply:
Oh.. Ok, then I think that we could follow this logic using `clearColorValue` for WebGPUBackend too. I believe that is what you are already planning?